contract/src/agoric-basics.contract.js

```diff
@@ -40,58 +40,112 @@ const bagCounts = bag => {
   const entries = getCopyBagEntries(bag);
   return entries.map(([_k, ct]) => ct);
 };
+
+/**
+ *
+ * @param {import('@endo/patterns').CopyBag} bag
+ * @param {Object.<string, {tradePrice: Amount, maxTickets: bigint}>} inventory
+ * @returns {boolean}
+ */
+export const hasInventory = (bag, inventory) => {
```

Comment:
_question_

Not sure if this is the best way to write unit test for helper functions like this?

Reply:
OTOH, I think `hasInventory` is probably better expressed by converting an inventory to an amount of tickets and then using `AmountMath.isGTE`.